src/controller/RaceController.js

```diff
@@ -0,0 +1,55 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import RaceView from '../view/RaceView.js';
+import CarModel from '../model/CarModel.js';
+import RaceModel from '../model/RaceModel.js';
+import InputValidator from './InputValidator.js';
+import { FORWARD_THRESHOLD } from '../constants/constraints.js';
+
+class RaceController {
+  constructor() {
+    this.view = new RaceView();
+    this.race = null;
+  }
+
+  async startRace() {
+    const carNames = await this.view.getCarNames();
+    InputValidator.validateCarName(carNames);
+    const attemptCount = await this.view.getAttemptCount();
+    InputValidator.validateAttemptCount(attemptCount);
+
+    const cars = carNames.map((name) => new CarModel(name));
+    this.race = new RaceModel(cars);
+
+    this.runRace(attemptCount);
+    this.showWinners();
+  }
```

Comment:
startRace에서 입력과 검증 로직이 같이 묶여있어서 분리해도 괜찮지 않을까? 하는 생각이 들었어요 이렇게 바꿔보는건 어떨까요?

```suggestion
async startRace() {
    const carNames = await this.getValidatedCarNames();
    const attemptCount = await this.getValidatedAttemptCount();

    const cars = carNames.map((name) => new CarModel(name));
    this.race = new RaceModel(cars);

    this.runRace(attemptCount);
    this.showWinners();
  }

  async getValidatedCarNames() {
    const carNames = await this.view.getCarNames();
    InputValidator.validateCarName(carNames);
    return carNames;
  }

  async getValidatedAttemptCount() {
    const attemptCount = await this.view.getAttemptCount();
    InputValidator.validateAttemptCount(attemptCount);
    return attemptCount;
  }
  ```

Reply:
`startRace`가 좀 더 간편해지고 관심사 분리가 더 잘 되는 것 같아요
시간 내서 코드 리뷰 해주셔서 감사합니다!
정말 도움이 많이 되었습니다~🙇‍♀️